### PR TITLE
fix: list pagination error

### PR DESCRIPTION
**Pull request by nuanyang233**

### CODE REVIEW

**Review by Jack-Works — DISMISSED** [review]

**Review by guanbinrui — APPROVED** [review]

File: packages/maskbook/src/plugins/Wallet/hooks/useCollectibles.ts [hunk 1]
```diff
@@ -28,28 +28,33 @@ export function useCollectibles(address: string, provider: CollectibleProvider,
...
+                            }),
+                        ),
+                    ],
+                    (a, b) => isSameAddress(a.address, b.address) && a.tokenId === b.tokenId,
+                ),
+            )
+            return values.current
+        } catch (e) {
+            if (values.current.length === 0) throw e
+            return values.current
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/Wallet/hooks/useCollectibles.ts:57]: looks strange but ok for now

**Review by Jack-Works — APPROVED** [review]

**Comment by Jack-Works** [comment]
@nuanyang233 Some checks were not successful

**Comment by guanbinrui** [comment]
> @nuanyang233 Some checks were not successful

FIXED.

**Review by guanbinrui — APPROVED** [review]

**Review by septs — COMMENTED** [review]
why based on tab index?

**Review by septs — APPROVED** [review]

File: packages/maskbook/src/plugins/Wallet/hooks/useCollectibles.ts (hunk 1 above)
Comment (nuanyang233) [reply packages/maskbook/src/plugins/Wallet/hooks/useCollectibles.ts:66]: There is the hope that the previous data will be retained after the page turning request reports an error

**Pull request merged**